tests/manual/desktop/HostServices/test.txt

```diff
@@ -0,0 +1,2 @@
+TXT Test
+Test page for HostServices.showDocument.
```

Comment:
Minor: It doesn't show up in the UI, but this file doesn't have a newline at the end of file